hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -138,8 +138,8 @@
   vcs_cov_excl_files: ["{dv_root}/tools/vcs/common_cov_excl.el"]
 
   // Build-specific coverage cfg files for VCS.
-  default_vcs_cov_cfg_file: "-cm_hier {dv_root}/tools/vcs/cover.cfg"
-  cover_reg_top_vcs_cov_cfg_file: "-cm_hier {dv_root}/tools/vcs/cover_reg_top.cfg"
+  default_vcs_cov_cfg_file: "-cm_hier {dv_root}/tools/vcs/cover.cfg -cm_common_hier"
```

Comment:
It is odd to have an extra switch appended at the end of each of these. Perhaps think of a better way?

Reply:
I see, thanks Sri, let me try that :）